CHANGES.md

```diff
@@ -0,0 +1,51 @@
+ringpop-node release notes
+==========================
+
+10.15.0
+-------
+
+**Feature: Parition Healing**
+* Partition healing implementation [#272](https://github.com/uber/ringpop-node/pull/272) [#273](https://github.com/uber/ringpop-node/pull/273)
+* Refactor in preperation for partition healing [#260](https://github.com/uber/ringpop-node/pull/260) [#259](https://github.com/uber/ringpop-node/pull/259) [#258](https://github.com/uber/ringpop-node/pull/258) [#266](https://github.com/uber/ringpop-node/pull/266)
+* Use a DiscoverProvider-function to discover the bootstrap hosts [#253](https://github.com/uber/ringpop-node/pull/253)
+* Deprecate "admin/reload"-endpoint [#255](https://github.com/uber/ringpop-node/pull/255)
+
+**Feature: Bi-directional full syncs [#251](https://github.com/uber/ringpop-node/pull/251)**
```

Comment:
Not sure, partition healing has the same properties as bidir full syncs and is a feature to me and not a bug fix.